packages/flutter/test/material/scrollbar_theme_test.dart

```diff
@@ -192,6 +192,111 @@ void main() {
     }),
   );
 
+  testWidgets('ScrollbarTheme can disable gestures', (WidgetTester tester) async {
+    final ScrollController scrollController = ScrollController();
+    await tester.pumpWidget(MaterialApp(
+      theme: ThemeData(scrollbarTheme: const ScrollbarThemeData(interactive: false)),
+      home: Scrollbar(
+        isAlwaysShown: true,
+        controller: scrollController,
+        child: SingleChildScrollView(
+          controller: scrollController,
+          child: const SizedBox(width: 4000.0, height: 4000.0)
+        ),
+      ),
+    ));
+    await tester.pumpAndSettle();
+    // Idle scrollbar behavior
+    expect(
+      find.byType(Scrollbar),
+      paints..rrect(
+        rrect: RRect.fromRectAndRadius(
+          const Rect.fromLTRB(790.0, 0.0, 798.0, 90.0),
```

Comment:
It would be helpful if  this was a constant var that had a comment which explained where the dimensions come from.